ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml

```diff
@@ -96,6 +100,31 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
+- as: aws-ipi-byo-route53-upgrade-rollback-oldest-supported-f28
+  cron: 25 21 14 * *
+  steps:
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:oldest-supported-z-rollback
+      OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
```

Comment:
Because we are setting `upgrade-edge`, so `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` is not a MUST.

Reply:
Removed. Thanks.